packages/scaffolding-cli/templates/src/ssr/state-management/index.ts

```diff
@@ -1,50 +1,49 @@
-import {Store, createStore, applyMiddleware, compose} from "redux"
+import {createWrapper} from "next-redux-wrapper"
+import {applyMiddleware, createStore, Store} from "redux"
+import logger from "redux-logger"
 import createSagaMiddleware, {Task} from "redux-saga"
-import getConfig from "next/config"
 import rootReducer from "./root-reducer"
 import rootSaga from "./root-saga"
-import logger from "redux-logger"
 
-const IS_BROWSER = typeof window !== "undefined"
+// const IS_BROWSER = typeof window !== "undefined"
 
-const {
-    publicRuntimeConfig: {NODE_ENV},
-} = getConfig()
+// const {
+//     publicRuntimeConfig: {NODE_ENV},
+// } = getConfig()
 
-const hasDevTools =
-    NODE_ENV === "development" && IS_BROWSER && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+// const hasDevTools = false
+// NODE_ENV === "development" && IS_BROWSER && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
```

Comment:
Remove the commented out lines if not needed?

Reply:
Done